.github/workflows/version-patch.yml

```diff
@@ -0,0 +1,154 @@
+        type: string
+
+      commit-5:
+        required: false
+        description:
+          'Optionally specify a 5th commit SHA to be included in the release.'
+        default: '########################################'
+        type: string
+
+jobs:
+  version:
+    runs-on: ubuntu-latest
+    if:
+      ${{ github.repository == 'carbon-design-system/carbon' &&
+      github.event.inputs.tag != 'v11.x' && github.event.inputs.existing-tag !=
+      'v11.x' }}
+    timeout-minutes: 60
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          fetch-depth: '0'
+          fetch-tags: true
+          ref: ${{ github.event.inputs.existing-tag }}
+      - name: Use Node.js version from .nvmrc
+        uses: actions/setup-node@cdca7365b2dadb8aad0a33bc7601856ffabcc48e # v4.3.0
+        with:
+          node-version-file: '.nvmrc'
+          registry-url: 'https://registry.npmjs.org'
+      - name: Create new release branch
+        run: |
+          git checkout -b ${{ github.event.inputs.tag }}
+      - uses: actions/cache@5a3ec84eff668545956fd18022155c47e93e2684 # v4.2.3
+        id: cache
+        with:
+          path: |
+            node_modules
+            */**/node_modules
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+      - name: Cherry pick commit-1
+        if:
+          ${{ github.event.inputs.commit-1 !=
+          '########################################'}}
+        run: |
+          git cherry-pick ${{ github.event.inputs.commit-1 }}
+      - name: Cherry pick commit-2
+        if:
+          ${{ github.event.inputs.commit-2 !=
+          '########################################'}}
+        run: |
+          git cherry-pick ${{ github.event.inputs.commit-2 }}
+      - name: Cherry pick commit-3
+        if:
+          ${{ github.event.inputs.commit-3 !=
+          '########################################'}}
+        run: |
+          git cherry-pick ${{ github.event.inputs.commit-3 }}
+      - name: Cherry pick commit-4
+        if:
+          ${{ github.event.inputs.commit-4 !=
+          '########################################'}}
+        run: |
+          git cherry-pick ${{ github.event.inputs.commit-4 }}
+      - name: Cherry pick commit-5
+        if:
+          ${{ github.event.inputs.commit-5 !=
+          '########################################'}}
+        run: |
+          git cherry-pick ${{ github.event.inputs.commit-5 }}
+      - name: Install dependencies
+        run: yarn install --immutable --immutable-cache --check-cache
+      - name: Update telemetry.yml
+        run: yarn run telemetry:config
+      - name: Create patch release
+        run: |
+          yarn lerna version patch --no-git-tag-version --no-push --yes
+      - name: Create release commit
+        run: |
+          git add -A
+          git commit -m 'chore(release): ${{github.event.inputs.tag}}'
```

Comment:
Should double quotes be used instead or will the shell still interpolate the variable with single quotes?